thrift/src/main/thrift/atoms/quiz.thrift

```diff
@@ -17,7 +17,6 @@ struct ResultGroup {
 
 struct Asset {
   1: required string type
-  /* what type is this? currently assuming opaque json */
   2: required shared.OpaqueJson data
```

Comment:
The type is clear. `shared.OpaqueJson` is an alias for a `string` defined in shared.thrift.